deploy-manage/deploy/cloud-enterprise/install-ece-procedures.md

```diff
@@ -0,0 +1,17 @@
+---
+navigation_title: Installation procedures
+applies_to:
+  deployment:
+    ece: all
+---
+
+# ECE installation procedures
+
+Choose the Elastic Cloud Enterprise deployment scenario that best fits your business needs:
+
+* [Deploy a small installation](deploy-small-installation.md): For development, test, and small-scale use cases.
+* [Deploy a medium installation](deploy-medium-installation.md): For many production setups.
+* [Deploy a large installation](deploy-large-installation.md): For deployments with significant overall search and indexing throughput.
+* [Deploy using Podman](fresh-installation-of-ece-using-podman-hosts.md): Fresh installation of ECE using Podman hosts.
+
+For installations using Podman instead of Docker, refer to [](./fresh-installation-of-ece-using-podman-hosts.md)
```

Comment:
Can we link to the [migration topic](https://docs-v3-preview.elastic.dev/elastic/docs-content/pull/593/deploy-manage/deploy/cloud-enterprise/migrate-ece-to-podman-hosts) in this overview so people know what it's for? 

do you think it belongs under here, or is it more of a sibling to "add more hosts"?

Reply:
It makes all sense, good catch. And considering RHEL moved from Docker to Podman and that forces all existing installations based on RHEL to migrate at some point I think it's really worthy to add the extra visibility.